aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/FlutterFactory.kt

```diff
@@ -0,0 +1,33 @@
+package com.amplifyframework.auth.cognito.helpers
+
+import com.amplifyframework.auth.AWSCredentials
+import com.amplifyframework.auth.cognito.AWSCognitoAuthSession
+import com.amplifyframework.auth.cognito.AWSCognitoUserPoolTokens
+import com.amplifyframework.auth.result.AuthSessionResult
+
+/**
+ * Helper class that allows Amplify Flutter to create necessary return types with internal constructors.
```

Comment:
See https://github.com/aws-amplify/amplify-swift/blob/7f094a9e88612b5b8fda1bce01ce71e7f1915053/Amplify/Categories/DataStore/Model/Internal/Schema/ModelSchema%2BDefinition.swift#L11-L12 for example message from Swift

Reply:
Also, "should not be considered" is probably fragmentary. :)